spec/features/user_upvotes_on_review_spec.rb

```diff
@@ -7,37 +7,34 @@
 } do
 
   describe 'User upvotes on a review' do
-    scenario 'user clicks upvote' do
+    scenario 'user clicks upvote', js: true do
       review = FactoryGirl.create(:review)
       beach = review.beach
       user = review.user
 
       sign_in(user)
 
-      visit '/'
-      click_on beach.name
+      visit beach_path(beach)
+
       click_on 'Upvote'
+      sleep 1
 
-      expect(page).to have_content("Upvote created successfully.")
-      expect(page).to_not have_content("Upvote not created successfully.")
-      review.reload
       expect(review.score).to eq(1)
     end
-    scenario 'user clicks remove upvote' do
+    scenario 'user clicks remove upvote', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.